doc/rtd/development/packaging.rst

```diff
@@ -0,0 +1,105 @@
+.. _building_and_distribution:
+
+Building and Distribution
```

Comment:
Nit: this page is about downstream packaging, not distribution of packages. I would recommend: "Downstream packaging", and same for the ref name.

Reply:
+1